opencog/scm/opencog/ure/ure-utils.scm

```diff
@@ -537,6 +539,26 @@
     (cog-set-atomspace! current-as)
     clean-weighted-rules))
 
+(define*-public (ure-search-rules  #:optional (as (cog-atomspace)))
+"
+  Retrieve all potential rules in the given or default atomspace.
+  https://github.com/singnet/ure/issues/16
```

Comment:
Ideally, you'd add a usage, cause doing `(help <function>)` only shows the comment, not the signature.